meta-vesnin/recipes-kernel/linux/linux-aspeed/0001-ARM-dts-aspeed-Add-YADRO-VESNIN-BMC.patch

```diff
@@ -0,0 +1,270 @@
+From 99c53c63a485a7cf929e0bae9e1db2e6a288153b Mon Sep 17 00:00:00 2001
+From: Alexander Filippov <a.filippov@yadro.com>
+Date: Tue, 19 Feb 2019 15:02:06 +0300
+Subject: [PATCH] ARM: dts: aspeed: Add YADRO VESNIN BMC
+
+VESNIN is an OpenPower machine with an Aspeed 2400 BMC SoC manufactured
+by YADRO.
```

Comment:
I'd say:
> VESNIN is a quad OpenPower P8 machine manufactured by YADRO.
> The machine uses an AST2400 BMC SoC.

The same applies to the commit log in this pull request itself.

Reply:
It has been sent to the Linux mainline with this commit message, so I propose keeping it as is.
But I will change the commit message in this pull request.